internal/oci/uvm.go

```diff
@@ -362,6 +396,7 @@ func SpecToUVMCreateOpts(ctx context.Context, s *specs.Spec, id, owner string) (
 		wopts.ProcessorWeight = ParseAnnotationsCPUWeight(ctx, s, annotationProcessorWeight, wopts.ProcessorWeight)
 		wopts.StorageQoSBandwidthMaximum = ParseAnnotationsStorageBps(ctx, s, annotationStorageQoSBandwidthMaximum, wopts.StorageQoSBandwidthMaximum)
 		wopts.StorageQoSIopsMaximum = ParseAnnotationsStorageIops(ctx, s, annotationStorageQoSIopsMaximum, wopts.StorageQoSIopsMaximum)
+		handleAnnotationFullyPhysicallyBacked(ctx, s.Annotations, wopts)
```

Comment:
These `handleAnnotation*` functions alter the values that are already set. For example, `handleAnnotationFullyPhysicallyBacked` annotation will set the value of `RootFSFile` to `uvm.InitrdFile` even if the original value is set to something else. I know we have a function that verifies the options but it would be good to move all these `handleAnnotation` functions at the end of their corresponding `if/else` blocks and then have a comment there which says that the order in which these functions are called is important.

Reply:
Updated, let me know if the comment is clear.